public/app.js

```diff
@@ -0,0 +1,70 @@
+{
+  const XNav = {
+    template: `
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="/100commitow-projekt" v-html="$root.env.TITLE"></a>
```

Comment:
żeby link zadziałał, katalog nie może być `/public`, a powinien być bezpośrednio `/` lub `/docs`

1) to wymóg gh-pages przy konfiguracji gh-pages (dyskusja tutaj https://github.com/orgs/community/discussions/23073#discussioncomment-4143213, docs https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#about-publishing-sources).

2) można skorzystać z custom workflow, ale jest w beta
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow